dynamic-checkbox-item/load_slack_channels_in_form_question.js

```diff
@@ -0,0 +1,80 @@
+var IGNORED_CHANNELS = [
+  'chat',
+  'slack-questions'
+]
+var FORM_ID = ''
+var SLACK_API_CHANNELS_URL = ''
+
+function onOpen () {
+  var form = FormApp.openById(FORM_ID)
+  var requestResponse = UrlFetchApp.fetch(SLACK_API_CHANNELS_URL)
+  var parsedJsonResponse = JSON.parse(requestResponse.getContentText())
+  var isSlackResponseOk = parsedJsonResponse.ok
+
+  if (!isSlackResponseOk) return
+
+  var eligibleChannels = prepareEligibleChannels(parsedJsonResponse)
+
+  var checkBoxItem = sanitizedCheckBoxItem(form)
+
+  eligibleChannels = eligibleChannels.map(function (channel) {
+    return checkBoxItem.createChoice(channel.name)
+  })
+
+  checkBoxItem
+    .setTitle('Select the channels you will probably join:')
+    .setChoices(eligibleChannels)
+}
+
+function prepareEligibleChannels (parsedJsonResponse) {
+  return parsedJsonResponse.channels.reduce(function (filteredChannels, channel) {
+    if (isChannelEligible(channel)) {
+      filteredChannels.push({
+        name: channel.name,
+        topic: channel.topic.value
+      })
+    }
+    return filteredChannels
+  }, [])
```

Comment:
@alexstyl This is the initial value in `.reduce` function, which represents an empty array.